### PR TITLE
Add LESS recipe

### PR DESCRIPTION
**Pull request by Integrayshaun**
## What changed
- [x] Add recipe for integrating Less

### CODE REVIEW

**Comment by Integrayshaun** [comment]
less (1)

Logo for recipe icon

File: src/content/recipes/less.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+</div>
+
+First of all, install the required dependencies:
+
+```shell
+yarn add -D @storybook/addon-styling less
+```
+
+Then add register `@storybook/addon-styling` in your `main.js` like so:
```
Comment (domyen) [inline src/content/recipes/less.md:33]:
```suggestion
Then register `@storybook/addon-styling` in your `main.js` like so:
```

File: src/content/recipes/less.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+```js
+module.exports = {
+  stories: ['../stories/**/*.mdx', '../stories/**/*.stories.@(js|jsx|ts|tsx)'],
+  addons: [
+    '@storybook/addon-essentials',
+    {
+      name: '@storybook/addon-styling',
+      options: {
+        less: {
+          // Require your Sass preprocessor here
```
Comment (domyen) [inline src/content/recipes/less.md:44]: Is this supposed to say Less preprocessor?

**Review by domyen — CHANGES_REQUESTED** [review]
Change Css » CSS
image

You might need to add yourself to contributors?

**Comment by Integrayshaun** [comment]
I'll have to look into the API because that's what seems to be making the CSS tag appear as Css

**Pull request merged**